WEB_SERVICE/DNS/_MEDIUM/DANGLING_DOMAINS/description.md

```diff
@@ -0,0 +1,100 @@
+Dangling domains occur when DNS records point to resources that are no longer active or under the organization's control. These misconfigurations can lead to domain hijacking, data leakage, and reputation damage. The following areas are key concerns in dangling domain management:
+
+## 1. Cloud Resource References
```

Comment:
I guess the titles will look too big. No?
![image](https://github.com/user-attachments/assets/a6e803a6-5046-4663-ba3f-0f61b3ae6f77)